alerta/auth/oidc.py

```diff
@@ -114,7 +115,7 @@ def openid():
         group_claim: userinfo.get(group_claim) or id_token.get(group_claim, []),
     }
 
-    login = username or nickname or email
+    login = email
```

Comment:
This is required to support OpenID connect providers that don't supply any of `username`, `nickname` or `email`. The Alerta auth flow needs at least one but only relying on email will definitely fail for lots of people.